### PR TITLE
added missing code

### PR DESCRIPTION
**Pull request by andregeorgi**

### CODE REVIEW

**Review by aleneagu96 — APPROVED** [review]

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -95,21 +95,23 @@ const Card = (props) => {
...
-              {t('priority')} <i>{priority ?? t('noPriority')}</i>
+              {t("priority")}: <i>{priority ?? t("no_priority")}</i>
             </Typography>
             <Typography variant="h5" component="h2">
-              {t('todo')}<i>{id ?? t('noId')}</i>
+              {t("todo")} #<i>{id ?? t("noId")}</i>
             </Typography>
             <Typography className={classes.pos} color="textSecondary">
-              {description ?? t('noDescription')}
+              {description ?? "No description"}
```
Comment (sorincretescu) [inline src/components/Card/Card.jsx:104]: Why did you remove the translation here?

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -119,7 +121,7 @@ const Card = (props) => {
                     ))}
                   </ul>
                 ) : (
-                  <p>{t('noSubtasks')}</p>
+                  <p>{t("no_subtasks")}</p>
```
Comment (sorincretescu) [inline src/components/Card/Card.jsx:124]: And why change it from camelCase to underscore? Use the phrases like t('No subtasks')

**Review by sorincretescu — APPROVED** [review]
